firestore/ios/firestore-snippets-cpp/AppDelegate.mm

```diff
@@ -17,17 +17,25 @@
 #import "AppDelegate.h"
 #import <Firebase/Firebase.h>
 #include "firebase/app.h"
+#include "firebase/firestore.h"
+
+using firebase::App;
+using firebase::firestore::Firestore;
 
 @interface AppDelegate ()
 
 @end
 
 @implementation AppDelegate
 
-
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [FIRApp configure];
-  firebase::App::Create();
+  // [START get_firestore_instance]
+  // Make sure the call to `Create()` happens some time before you call Firestore::GetInstance().
+  App::Create();
+  Firestore * db = Firestore::GetInstance();
```

Comment:
Nit: this is one style of asterisk placement that isn't allowed by the style guide:
> you may place the asterisk/ampersand adjacent to either the type or the variable name

I'd suggest attaching it to the type, i.e. `Firestore*`, as this is the style we use in the Firestore SDK, and I believe the style used by other Firebase C++ SDKs as well.

Reply:
Fixed.